util/packager.js

```diff
@@ -1,82 +1,129 @@
+      const jsFile = file.replace(".json", ".mjs");
+      writeFileSync(join(exportBase, jsFile), updatedContents);
+    } catch (e) {
+      console.log(e);
+
       continue;
     }
     // Replace the base64 string in the corresponding library .fsh file
-    const contentBase64 = Buffer.from(contentString).toString('base64');
-    const fshFile = file.replace('.json','.fsh');
+    const contentBase64 = Buffer.from(contentString).toString("base64");
+    const fshFile = file.replace(".json", ".fsh");
     try {
-      const outputContents = readFileSync(fshPath + sep + fshFile).toString();
-      const updatedContents = outputContents.replace(
-        /\* content\[\=\]\.data \= \".+\"/,
-        '* content[=].data = \"' + contentBase64 + '\"'
+      let outputContents = readFileSync(fshPath + sep + fshFile).toString();
+      let lines = outputContents.split("\n");
+
+      // Find the index of the line starting with '* description'
+      const descriptionIndex = lines.findIndex((line) =>
+        line.trim().startsWith("* description")
       );
+      if (descriptionIndex !== -1) {
+        lines = lines.slice(0, descriptionIndex + 1);
+      }
+
+      const updatedContents = `${lines.join('\n')}
+* content[cqlContent].contentType = urn:ietf:bcp:13#text/cql "text/cql"
+* content[cqlContent].data = "${cql.toString("base64")}"
+* content[1].contentType = urn:ietf:bcp:13#application/elm+json "application/elm+json"
+* content[1].data = "${contentBase64}"
+
+`;
+
       writeFileSync(fshPath + sep + fshFile, updatedContents);
-    } catch(e) {
+    } catch (e) {
       continue;
     }
     // Replace the base64 string in the corresponding library .json file
-    const jsonFile = 'Library-' + file;
+    const jsonFile = "Library-" + file;
     try {
-      const outputContents = readFileSync(fhirPath + sep + jsonFile).toString();
-      const updatedContents = outputContents.replace(
-        /\"data\": \".+\"/,
-        '\"data\": \"' + contentBase64 + '\"'
+      const outputContents = JSON.parse(
+        readFileSync(fhirPath + sep + jsonFile).toString()
       );
-      writeFileSync(fhirPath + sep + jsonFile, updatedContents);
-    } catch(e) {
+      outputContents.content = [
+        {
+          contentType: "text/cql",
+          data: cql.toString("base64"),
+        },
+        {
+          contentType: "application/elm+json",
+          data: contentBase64,
+        },
+      ];
+
+      writeFileSync(
+        fhirPath + sep + jsonFile,
+        JSON.stringify(outputContents, null, 2)
+      );
+    } catch (e) {
       continue;
     }
   }
 }
 
 // Loop over all generated FHIR resources and convert them to js files.
 files = readdirSync(fhirPath);
-exportBase = join('dist','fhir');
-if (!existsSync(exportBase)) mkdirSync(exportBase, {recursive: true});
+exportBase = join("dist", "fhir");
+const bundleFilepath = bundlePath + sep + "transactionBundle.json";
```

Comment:
I wonder if we should also place this transaction bundle with the rest of the exported resources in `dist/`?

Reply:
I don't think so, it's more for convenience than it is a real bundle resource.